src/home/scrolls.vue

```diff
@@ -9,18 +9,18 @@
   h6.middle-align
     span Chips
   nav.scroll
```

Comment:
This scroll its very close to the element maybe we can add some padding ?

Reply:
![image](https://github.com/beercss/beercss/assets/12979016/1715e15d-e5df-4c9b-ac22-eb15ba946c32)
![image](https://github.com/beercss/beercss/assets/12979016/294d2f59-56f6-4fb2-9a21-3f7377573eee)

For me it's ok in both cases, do you prefer with `small-padding`?